doc/how-to-contribute.rst

```diff
@@ -126,32 +130,48 @@ Make sure the documentation is updated
   * Readthedocs *should* automatically build the docs for each:
       - push to develop (latest docs)
       - push to stable (stable docs)
-      - new tags (e.g v9.2.x)
+      - new tags (e.g v9.3.1)
   * *But*, the webhooks are somehow broken, so it probably won't work automatically!
       - Trigger the builds manually here:  https://readthedocs.org/projects/pytango/builds/
       - Set the new version to "active" here:
         https://readthedocs.org/dashboard/pytango/versions/
 
 Create a release tag on GitHub
   * On the Releases page, use "Draft a new release".
-  * Tag must match the format of previous tags, e.g. ``v9.2.6``.
+  * Tag must match the format of previous tags, e.g. ``v9.3.1``.
   * Target must be the ``stable`` branch.
 
 Upload the new version to PyPI
   * Log in to https://pypi.org.
   * Get account permissions for PyTango from another contributor, if necessary.
   * If necessary, pip install twine: https://pypi.org/project/twine/)
-  * Build update and upload, from the the ``stable`` branch:
+  * Build release from the tagged commit:
       - ``$ git clean -xfd  # Warning - remove all non-versioned files and directories``
+      - ``$ git fetch``
+      - ``$ git checkout v9.3.1``
       - ``$ python setup.py sdist``
-      - ``$ twine upload dist/pytango-9.2.N.tar.gz``
+  * Optional:  Upload to https://test.pypi.org, and make sure all is well:
+      - ``$ twine upload -r testpypi dist/pytango-9.3.1.tar.gz``
+  * Optional:  Test installation (in a virtualenv):
+      - ``$ pip install -i https://test.pypi.org/simple/ pytango``
+  * Upload the source tarball to the real PyPI:
+      - ``$ twine upload dist/pytango-9.3.1.tar.gz``
+  * Run build for the tag on AppVeyor, download artifacts, and upload wheels:
+      - ``$ twine upload dist/pytango-9.3.1-cp27-cp27m-win32.whl``
+      - ``$ twine upload dist/pytango-9.3.1-cp27-cp27m-win_amd64.whl``
+      - ``$ twine upload dist/pytango-9.3.1-cp36-cp36m-win32.whl``
+      - ``$ twine upload dist/pytango-9.3.1-cp36-cp36m-win_amd64.whl``
+      - ``$ twine upload dist/pytango-9.3.1-cp37-cp37m-win32.whl``
+      - ``$ twine upload dist/pytango-9.3.1-cp37-cp37m-win_amd64.whl``
```

Comment:
@tiagocoutinho Are we just going upload the wheels?

Reply:
> @tiagocoutinho Are we just going upload the wheels?

Yes, do you have something else in mind?

Reply:
I was asking because I see the AppVeyor build artefacts also have some .exe files.  I'm happy with just the wheels.